image/sigmoid/sigmoid.py

```diff
@@ -0,0 +1,22 @@
+import tensorflow as tf
```

Comment:
Always good to add a header block. Block would include authorship, license, copyright, dates and description of what the driver does.